torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -336,6 +338,13 @@ struct ConstantPropagator {
   }
 
   void ConstantPropagation(Node* n) {
+    auto allowed = JIT_BISECT();
+    if (!allowed) {
+      JIT_BISECT_LOG(
```

Comment:
This will probably spam the output after the optimization has reached its limit (i.e. we will keep trying to run the optimization no matter what, but with opt limit after some point each run would simply print this message and return). However, it might useful to detect the **last** invocation of the optimization - would that be possible to do somehow?